transform-plugins/README.rst

```diff
@@ -102,111 +94,124 @@ JSON Parser
   Realtime
 :Description:
-  Formats a structured record as JSON Object. Plugin will convert the structured record to JSON object and write to output record. The output record should have only one field of type STRING or BYTE array.
+  Formats a Structured Record as JSON Object. Plugin will convert the Structured Record to a JSON object and write to the output record. The output record schema is a single field, either type STRING or type BYTE array.
 :Configuration:
-    **schema:** Specifies the output schema that includes only one of type STRING or BYTE array. 
+  **schema:** Specifies the output schema, a single field either type STRING or type BYTE array.
 
 Clone Record
 -------------
-
-:Id:
-  CloneRecord
+:ID:
+  **CloneRecord**
 :Type:
   Transform
 :Mode:
   Batch and
   Realtime
 :Description:
-  Makes copy of every input record received configured number of times on the output. This transform does not change any record fields or types. It's identity. 
+  Makes a copy of every input record received for a configured number of times on the output. This transform does not change any record fields or types. It's an identity transform.
 :Configuration:
-    **copies:** Specifies numbers of copies of input record that has be emitted. 
+  **copies:** Specifies the numbers of copies of the input record that are to be emitted.
 
 Stream Formatter
 -------------
-
-:Id:
-  StreamFormatter
+:ID:
+  **StreamFormatter**
 :Type:
   Transform
 :Mode:
   Batch and
   Realtime
 :Description:
-  Formats a structured record as Stream format. Plugin will convert the structured record to Stream format. It will include header configuration and body configuration. The body of Stream event can be of any of the two types : CSV or JSON.
+  Formats a Structured Record as Stream format. Plugin will convert the Structured Record to Stream format.
+  It will include a header and body configurations. The body of the Stream event can be either type CSV or JSON.
 :Configuration:
-    **body:** Specifies the fields from input structured record that should be included in the body of Stream event. 
-    **header:** Specifies the fields from input structured record that should be included in the header of Stream event. 
-    **format:** Specifies the format of the body. Currently supported formats are JSON, CSV, TSV and PSV.
-    **schema:** Specifies the output schema. The output schema can have only two fields. One of type STRING and the other of type MAP<STRING, STRING>.
+  - **body:** Specifies the input Structured Record fields that should be included in the body of the Stream event.
+  - **header:** Specifies the input Structured Record fields that should be included in the header of the Stream event.
+  - **format:** Specifies the format of the body. Currently supported formats are JSON, CSV, TSV, and PSV.
+  - **schema:** Specifies the output schema. The output schema can have only two fields: one of type STRING and the other of type MAP<STRING, STRING>.
```

Comment:
I want to use my field names rather than trying to understand what is the name I should be using.